packages/shared/package.json

```diff
@@ -15,6 +15,10 @@
       "types": "./dist/index.d.ts",
       "default": "./dist/index.js"
     },
+    "./rspack": {
```

Comment:
I need to import `rspack.RuntimeGlobals`. I don't know where this place is correct